demos/rogers/tests/tests.py

```diff
@@ -39,18 +39,14 @@ class TestRogers(object):
 
         autobots = 36
 
-        sandbox_output = subprocess.check_output(
-            "dallinger sandbox",
-            shell=True)
+        sandbox_output = subprocess.check_output(["dallinger", "sandbox"])
 
         m = re.search('Running as experiment (.*)...', sandbox_output)
         exp_id = m.group(1)
         url = "http://" + exp_id + ".herokuapp.com"
 
         # Open the logs in the browser.
-        subprocess.call(
-            "dallinger logs --app " + exp_id,
-            shell=True)
+        subprocess.call(["dallinger", "log", "--app", exp_id])
```

Comment:
I think `call` should be fine? If for some reason the call failed and the logs didn't open, it wouldn't invalidate the test. (I will also not that this experiment-specific test isn't included in the test suite run on the CI server, and will soon be rewritten using the up-and-coming bot framework.)